specification/components/schemas/extensions/ServiceSummaryViewExtension.yaml

```diff
@@ -0,0 +1,473 @@
+            valueCodeableConcept:
+              type: object
+              required:
+                - coding
+              properties:
+                coding:
+                  type: array
+                  minItems: 1
+                  maxItems: 1
+                  items:
+                    type: object
+                    required:
+                     - system
+                     - code 
+                    properties:
+                      system:
+                        type: string
+                        enum:
+                          - https://fhir.nhs.uk/STU3/CodeSystem/eRS-CapacityAlert-1
+                        example: 'https://fhir.nhs.uk/STU3/CodeSystem/eRS-CapacityAlert-1'
+                      code:
+                        type: string
+                        enum:
+                          - LOW
+                        example: 'LOW'
+        - title: Whether the service is Directly or Indirectly Bookable
+          type: object
+          required:
+            - url
+            - valueCodeableConcept
+          properties:
+            url:
+              type: string
+              enum:
+                - bookableType
+              example: 'bookableType'
+            valueCodeableConcept:
+              type: object
+              required:
+                - coding
+              properties:
+                coding:
+                  type: array
+                  minItems: 1
+                  maxItems: 1
+                  items:
+                    type: object
+                    required:
+                     - system
+                     - code 
+                    properties:
+                      system:
+                        type: string
+                        enum:
+                          - https://fhir.nhs.uk/STU3/CodeSystem/eRS-BookableType-1
+                        example: 'https://fhir.nhs.uk/STU3/CodeSystem/eRS-BookableType-1'
+                      code:
+                        type: string
+                        enum: 
+                          - DIRECTLY_BOOKABLE
+                          - INDIRECTLY_BOOKABLE
+                          - UNSPECIFIED
+                        example: 'DIRECTLY_BOOKABLE'
+        - title: Link to NHS website providing information about the Service such as contact details, map, department, facilities and reviews.
+          type: object
+          required:
+            - url
+            - valueString
+          properties:
+            url:
+              type: string
+              enum:
+                - linkToNHSWebsite
+              example: 'linkToNHSWebsite'
+            valueString:
+              type: string
+              example: 'https://www.nhs.uk/service-search/other-services/chooseandbook?serviceId=11002'
+        - title: Service location
+          type: object
+          required:
+            - url
+            - valueReference
+          properties:
+            url:
+              type: string
```

Comment:
I think this property should have a nested object to model FHIR's Reference resource. For services we populate the "identifier" attribute of a reference using the system "https://directory.spineservices.nhs.uk/STU3/Organization" with the "value" attribute detailing the ODS code.